components/textbox/textbox.jsx

```diff
@@ -242,7 +243,7 @@ export default class Textbox extends React.PureComponent {
                     onBlur={this.handleBlur}
                     onHeightChange={this.handleHeightChange}
                     style={{visibility: this.props.preview ? 'hidden' : 'visible'}}
-                    inputComponent={AutosizeTextarea}
+                    inputComponent={PostEditor}
```

Comment:
I'm worried that we are losing all custom code from the previous component, would this pass cypress or some of the QA tests for regression?